services/verification-service/src/verifications/dto.rs

```diff
@@ -60,11 +60,11 @@ impl IntoResponse for VerificationError {
 }
 
 // TODO deserialize is only required during controller tests - can we conditionally derive?
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize)]
 pub struct VerificationResponseDto {
     pub documents: Vec<Document>,
     pub credentials: Vec<SpecializedJsonCredential>,
-    pub results: Vec<String>,
+    pub results: Vec<VcVerificationResult>,
```

Comment:
Could we also add another field just called `verified` of type boolean that is true if all VerificationResults are valid?